cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
Shouldn't with your refactory, the methods defined here can be reused for both coreos and trusty, so that we can remove those duplicated code when introducing trusty at the first place?